rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+
+6. There will be a formal ceremony to onboard new members and graduate retiring members. Ceremonies will take place on video calls attended by all current team members and any interested members of the community. Nobody will be required to show their face or voice, but team members must listen and actively participate, even if only through text.
+
+  In exceptional circumstances it is permissible for team members not directly involved in the proceedings to miss them, but these authors do consider this ceremony to be an important ritual for establishing a coherent culture, and hope that team members will keep that in mind and make every effort to attend.
+  
+  The ceremony will be led by an existing team member who will introduce newcomers and announce retirements. The member leading the ceremony will explain the team's values and responsibilities, and how the community team operates. New members will then each, individually state: "I understand my responsibilities as a member of the community team and agree to uphold the community values." Following the formal ceremony, there will be celebratory social interaction.
+
+  This ceremony may seem extremely formal for a technical community, but it serves an important purpose. In a world with very little respect for ceremony, it's important to have things that bring us together, even if they are simple things. Since the community team's responsibility is an important one, it is appropriate for team members to treat it with gravity. It is likewise important to take time to celebrate happy things, so that shared experiences can become the basis of cooperation.
+
+## Authority of the community team
+[authority]: #authority
+
+Team members will have the authority to ban community members from any community social space, or from all such spaces. Team members will be added to all necessary access control lists to exercise this power, as soon as possible after the onboarding ceremony.
+
+Team members will manage their own access control lists, adding and removing each other on Discourse, GitHub, Mjolnir (the Matrix bot), and anywhere else that's necessary.
+
+Community social spaces include Disourse, GitHub, and Matrix, as well as any analogous spaces which may exist in the future. It is expected that team members will not act alone when implementing bans, except in situations which are both urgent and exceptional. Like all moderation actions, bans will be publicly disclosed to the community, and explained. Banning somebody from participation on GitHub does prevent them from submitting pull requests; it is thus equivalent to full expulsion from the technical effort which the community is dedicated to.
```

Comment:
Although the Discord is not "official", it is still a place where hundreds of members of the community are gathered.
It feels strange to leave it aside when trying to increase the safety and well-being of the community.

Reply:
I'd like to add that there is also the Telegram group, which if i remember correctly is unofficial, it is smaller than the Discord, though i still believe that it is significant.

Reply:
since the discord is not an official nixos social space, we don't feel entitled to try and take control in an official capacity. if the discord wants to become official under the jurisdiction of the community team, that's a separate issue entirely.

Reply:
What i mean is that the actual Nix/NixOS community is too spread out or decentralized for this change to be worth the effort or make a significant change. It will be too hard to have control over everything.

Reply:
IMHO, this RFC is less about getting complete control over everything that putting mechanisms in place to guarantee safe spaces.

Reply:
There isn't a need to be in control of everything. There are certain groups where it is meaningful to have values and moderation, both of which we already have today.

Reply:
The RFC could add wording suggesting that unofficial communities are invited to officially endorse the statement of values. Endorsement of these values I guess could go without getting the formal community team involved in their community.

These communities can further talk with members of the more formal community team if they feel the need. That is, without requiring them to become official.